framework/physics/SolverBase/chi_solver.cc

```diff
@@ -22,6 +22,8 @@ chi::InputParameters Solver::GetInputParameters()
 
   params.AddOptionalParameter("dt", 0.01, "Desired initial timestep size.");
   params.AddOptionalParameter("time", 0.0, "Current time of the solver.");
+  params.AddOptionalParameter(
+    "start_time", 1.0, "Transient start-time if applicable.");
```

Comment:
What is the logic with this default value being the same as `end_time`?

Reply:
Good catch, Thanks